src/components/sale/SalesLeadTable.tsx

```diff
@@ -501,6 +501,82 @@ const SalesLeadTable: React.VFC<{
 
   const selectedRowLeads = leads.filter(lead => selectedRowKeys.includes(lead.id))
 
+  const [managerId, setManagerId] = useState('')
+  const businessAppId = useRef('')
+
+  const handleTransfer = async () => {
+    const _managerId = window.prompt('你要轉移此名單給哪個承辦編號？')?.trim()
+    _managerId && setManagerId(_managerId)
+  }
+
+  const { data: memberAppId } = useQuery(GetMemberAppID, {
+    variables: { managerId: managerId },
+    skip: managerId === '',
+  })
+
+  const { data: managerAppIds } = useQuery(GetManagerAppIDList, {
+    variables: { managerIds: selectedRowKeys },
+    skip: selectedRowKeys.length === 0,
+  })
+
+  useEffect(() => {
+    if (memberAppId == null || managerId === '') return
+    const matchedAppIds: Record<string, string>[] = []
+    const nonMatchingAppIds: Record<string, string>[] = []
+
+    businessAppId.current = memberAppId?.member[0]?.app_id
+    if (businessAppId.current == null) {
+      alert(`此會員不存在: ${managerId}`)
+      setManagerId('')
+      return
+    }
+
+    for (const _managerId of managerAppIds.member) {
```

Comment:
Why use an underscore as the parameter prefix?

Reply:
Thanks for the suggestion! I've removed the underscores to make the code more readable. Appreciate your feedback!

Reply:
there are different types of naming convention
```
_variable
__variable
variable_
variable__
_variable_
__variable__
```